src/pipeline.py

```diff
@@ -0,0 +1,24 @@
+import sys
+
+import extract_entity
+import extract_topic
+
+
```

Comment:
remove extra line. Should only have two lines between new functions and classes.